hack/lib/init.sh

```diff
@@ -10,6 +10,7 @@ set -o nounset
 set -o pipefail
 
 API_GROUP_VERSIONS="\
+config/v1alpha1 \
```

Comment:
do we need that backslash in this string?

Reply:
actyally, no :+1: